indexeddb-examples/idbkeyrange/style/style.css

```diff
@@ -1,93 +1,73 @@
-/* || basic set up + sizing for containers */
-
 html,
 body {
-  margin: 0;
   font-family: sans-serif;
-  font-size: 100%/1.5 sans-serif;
-}
-
-html {
-  background-color: #55aa55;
-}
-
-body {
-  max-width: 800px;
-  margin: 0 auto;
-  padding: 30px 0;
+  background: linear-gradient(aquamarine, #9198e5);
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  margin: 0;
 }
 
-h1 {
+h1,
+p {
   text-align: center;
-  font-size: 4rem;
-  color: white;
-  text-shadow: -1px -1px 2px black;
+  font-size: 1.5rem;
 }
 
 ul {
+  background: white;
   text-align: center;
   list-style-type: none;
   padding: 2rem 1rem;
-  background: white
-    linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.2));
-  box-shadow: 3px 3px 10px black;
+  border: 2px solid black;
 }
 
 li {
-  font-size: 2rem;
+  font-size: 1.2rem;
   padding-bottom: 1rem;
 }
 
-form {
-  float: right;
-  width: 55%;
-}
-
-ul {
-  float: left;
-  width: 40%;
+#container {
+  margin: 1rem;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 2rem;
 }
 
 form {
-  font-size: 1.7rem;
+  font-size: 1.2rem;
   margin-top: 1rem;
 }
 
 form > div {
```

Comment:
```suggestion
form > div, form >fieldset>div  {
```

Reply:
done in https://github.com/mdn/dom-examples/pull/303/commits/1888fa0dfdade190a8258b239db758590b542b19